src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -80,25 +85,27 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 		}
 		return null;
 	}
-	
-	@SuppressWarnings("null")
+
 	@Override
-	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) {
+	@SuppressWarnings("null")
+	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
 		int a = delta == null ? 0 : ((Number) delta[0]).intValue();
 		final Object[] os = getExpr().getArray(e);
```

Comment:
lets call this `objects` and `object` as this is misleading as to what `os` is. Same for `convert` method.